.github/workflows/ruby.yml

```diff
@@ -29,14 +29,6 @@ jobs:
             gemfile: gemfiles/Gemfile-rails.6.0.x
         experimental: [false]
         include:
-          - ruby: 2.5
```

Comment:
Why did you remove these versions?

Reply:
@guillaumebriday Because Rails 7.0.0-alpha does not support ruby 2.5 and 2.6, only 2.7+ 👍

Reply:
I think so these removes can be done in other PR separately, only that change context to rails 7. What do you think about that?

Reply:
Yes I think it's should not be in this PR

Reply:
Considering that branch `5-x-stable` does not receive frequent changes, and that this PR is small enough to be understood well (despite CI fixes be together with adding of Yarn v3), can we merge it? 🍻

Reply:
@pedrofurtado sounds like the preferred way forward is for you to pull the workflow changes out of this PR, focus it 💯 on just bringing yarn v3 support in for a quick merge....

Reply:
PR splitted in two: this that provides yarn 3 support, and other that fixes CI ( #3106 )